todo_app/app.py

```diff
@@ -78,6 +89,10 @@ def new_item():
 
             new_card = ToDoCard(0, new_item_title, trello_default_list, due_date, description, datetime.datetime.today())
             create_todo_card(new_card)
+            if 'LOGIN_DISABLED' not in app.config:
+                app.logger.info(f"New card created by {current_user.id}")
+            else:
+                app.logger.info(f"New card created by TestUser")
         return redirect(request.headers.get('Referer'))
```

Comment:
It would be good to think briefly about what other types of things might be worth logging (and particularly at different levels); for example here if the user didn't have write permissions but was trying to write anyway it might be worth warning about (since they shouldn't be seeing the option to do that behaviour)